crypto/secp256k1/secp256.go

```diff
@@ -49,6 +49,41 @@ var (
 	ErrRecoverFailed       = errors.New("recovery failed")
 )
 
+func VerifySeckeyValidity(seckey []byte) error {
+	if len(seckey) != 32 {
+		return errors.New("priv key is not 32 bytes")
+	}
+	var seckey_ptr *C.uchar = (*C.uchar)(unsafe.Pointer(&seckey[0]))
+	ret := C.secp256k1_ec_seckey_verify(context, seckey_ptr)
+	if int(ret) != 1 {
+		return errors.New("invalid seckey")
```

Comment:
Since variable `ErrInvalidKey` has already been defined, `errors.New("invalid seckey")` can be replaced with `ErrInvalidKey`.